cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -362,3 +376,27 @@ def _pids(self, key, value):
         doi_identifier["provider"] = "external"
 
     return {"doi": doi_identifier}
+
+
+@model.over("contributors", "^710__")
+@for_each_value
+def corporate_author(self, key, value):
+    """Translates corporate author."""
+    if "g" in value:
+        contributor = {
+            "person_or_org": {
+                "type": "organizational",
+                "name": StringValue(value.get("g")).parse(),
+                "family_name": StringValue(value.get("g")).parse(),
```

Comment:
Why are we populating the same for family_name?

Reply:
because it is a required field and name is propagated based on family_name + name combination. in this case they are the same - it is an organisation